migrations/2023-06-06-104440_index_post_url/up.sql

```diff
@@ -1 +1,9 @@
+-- Make a hard limit of 512 for the post.url column
+-- Truncate existing long rows.
+update post set url = left(url, 512) where length(url) > 512;
```

Comment:
I tried to find the longest urls on lemmy.ml, but somehow the query `select length(url) from post order by length(url) desc limit 10;` is returning blank results. Anyway truncating sounds like a bad idea, why not use 1024 as limit?

Reply:
I did this locally. Found that only 81 out of 156k posts were over 512, and there were maxed out. 512 is probably good, anything more might blow up the index to support a very tiny numeber of posts.

Reply:
Alright